common/src/View/UVViewHelper.cpp

```diff
@@ -205,25 +205,23 @@ namespace TrenchBroom {
 
             const auto toTex = m_face->toTexCoordSystemMatrix(vm::vec2f::zero, vm::vec2f::one, true);
             const auto transformedVertices = toTex * m_face->vertexPositions();
```

Comment:
Rename to `texVertices`
Also add `const auto camVertices = toCam * m_face->vertexPositions();`

Reply:
Done